dipy/reconst/tests/test_sfm.py

```diff
@@ -2,6 +2,7 @@
 import numpy as np
 import numpy.testing as npt
 import pytest
+import time
```

Comment:
I don't think this is being used.

Reply:
yep, I had included it initially for benchmarking, but will now remove.

Reply:
yep, removed.